ssl/statem/extensions_clnt.c

```diff
@@ -989,7 +989,7 @@ EXT_RETURN tls_construct_ctos_psk(SSL *s, WPACKET *pkt, unsigned int context,
     SSL_SESSION_free(psksess);
     return ret;
 #else
-    return 1;
+    return EXT_RETURN_SENT;
```

Comment:
Surely this should be EXT_RETURN_NOT_SENT

Reply:
Right! In a sense I did intend to ask to double-check this. But do note that 1 is actually EXT_RETURN_SENT, so that returning EXT_RETURN_NOT_SENT would be different. Are we even fixing an actual bug here? Please, re-confirm...

Reply:
Yes, looks like a bug.

Reply:
Additional commit pushed. However! There is actually another place I was meaning to ask to double-check. Extra line comment is coming...